kalliope/core/ConfigurationManager/YAMLLoader.py

```diff
@@ -73,7 +73,7 @@ def __init__(self, file_path):
 
         # load the yaml file
         with open(file_path, "r") as f:
-            self.data = yaml.load(f)
+            self.data = yaml.full_load(f)
```

Comment:
If this is a new method, we should then precise the required version of the lib in our requirement file, dont you think?

Reply:
Of course, you are right, I pushed a to fix a minimun version of the pyyaml package